src/Tasks/ResolveManifestFiles.cs

```diff
@@ -380,6 +382,29 @@ private void GetOutputAssemblies(List<PublishInfo> publishInfos, List<ITaskItem>
                 {
                     if (!IsFiltered(item))
                     {
+                        // ClickOnce for .NET 4.X should not publish duplicate satellite assemblies.
+                        // This will cause ClickOnce install to fail. This can happen if some package
+                        // decides to publish the en-us resource assemblies for other locales also.
+                        if (!LauncherBasedDeployment && satelliteRefAssemblyMap.ContainsItem(item))
+                        {
+                            Debug.Assert(false, $"Duplicate satellite assembly '{item.ItemSpec}' skipped in _managedAssemblies");
+                            continue;
+                        }
+
+                        // If we get a resource assembly in managed references, determine whether to be publish it based on _targetCulture
```

Comment:
Would it be accurate to say

```suggestion
                        // apply the culture publishing rules to managed references as well as satellites
```

?

Reply:
updated with tweaks.